### ISSUE TITLE
This model's maximum context length is 8191 tokens, however you requested 89686 tokens (89686 in your prompt)

### ISSUE THREAD

**Issue by Nickonomic** [Significant-Gravitas/Auto-GPT#1639]
### Duplicates

- [X] I have searched the existing issues

### Steps to reproduce 🕹

The program is trying to process an absurd amount of information at once. It happens over and over again.

Adding chunk 17 / 20 to memory
SYSTEM:  Command browse_website returned: Error: This model's maximum context length is 8191 tokens, however you requested 89686 tokens (89686 in your prompt;
 0 for the completion). Please reduce your prompt; or completion length.

### Your prompt 📝

```yaml
# Paste your prompt here
```

**Comment by SlistInc** [comment]
I think this may be related to #796 and #1211?

**Comment by endolith** [comment]
No, this is not fixed as of a2e16695:

```
NEXT ACTION:  COMMAND = read_file ARGUMENTS = {'filename': './README.md'}
Enter 'y' to authorise command, 'y -N' to run N continuous commands, 'n' to exit program, or enter feedback for ...
Input:y
-=-=-=-=-=-=-= COMMAND AUTHORISED BY USER -=-=-=-=-=-=-=
Traceback (most recent call last):
  File "C:\Anaconda3\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Anaconda3\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "…\autogpt\__main__.py", line 5, in <module>
    autogpt.cli.main()
  File "C:\Anaconda3\lib\site-packages\click\core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "C:\Anaconda3\lib\site-packages\click\core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "C:\Anaconda3\lib\site-packages\click\core.py", line 1635, in invoke
    rv = super().invoke(ctx)
  File "C:\Anaconda3\lib\site-packages\click\core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "C:\Anaconda3\lib\site-packages\click\core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "C:\Anaconda3\lib\site-packages\click\decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "…\autogpt\cli.py", line 177, in main
    agent.start_interaction_loop()
  File "…\autogpt\agent\agent.py", line 213, in start_interaction_loop
    self.memory.add(memory_to_add)
  File "…\autogpt\memory\local.py", line 76, in add
    embedding = create_embedding_with_ada(text)
  File "…\autogpt\llm_utils.py", line 170, in create_embedding_with_ada
    return openai.Embedding.create(
  File "C:\Anaconda3\lib\site-packages\openai\api_resources\embedding.py", line 33, in create
    response = super().create(*args, **kwargs)
  File "C:\Anaconda3\lib\site-packages\openai\api_resources\abstract\engine_api_resource.py", line 153, in create
    response, _, api_key = requestor.request(
  File "C:\Anaconda3\lib\site-packages\openai\api_requestor.py", line 226, in request
    resp, got_stream = self._interpret_response(result, stream)
  File "C:\Anaconda3\lib\site-packages\openai\api_requestor.py", line 619, in _interpret_response
    self._interpret_response_line(
  File "C:\Anaconda3\lib\site-packages\openai\api_requestor.py", line 682, in _interpret_response_line
    raise self.handle_error_response(
openai.error.InvalidRequestError: This model's maximum context length is 8191 tokens, however you requested 11017 tokens (11017 in your prompt; 0 for the completion). Please reduce your prompt; or completion length.
```

I have a proposed fix in https://github.com/Significant-Gravitas/Auto-GPT/pull/2039

**Comment by Pwuts** [comment]
@endolith you are confusing different commands as using the same text processing. They don't. For `browse_website` and `get_text_summary`, this was fixed in #2542. For issues with other commands, please don't post here.
